test/runtests.jl

```diff
@@ -497,6 +497,23 @@ using Test
         @test net_space.branches.j == net_space_manual.branches.j
     end
 
+    @testset "End-of-line comments" begin
+        net_eol = parse_network("testfiles/eolcomments.raw")
+
+        # Only `Buses30` parse trailing EOL comments.
+        buses = net_eol.buses
+        @test length(buses) == 2
+        @test buses.i == [10010, 337918]  # first col
+        @test buses.owner == [1, 1]   # last col before comments
+        @test all(contains.(buses.comment, ["NOBO     1", "NAU_E2   1"]))
+
+        loads = net_eol.loads
+        @test length(loads) == 1
+        @test loads.i == [10010]  # first col
+        @test loads.owner == [1]  # last non-missing col
+        @test isequal(loads.intrpt, [missing]) # last col
```

Comment:
I'm wondering what exactly this test is checking for. Is it checking that the end of line comment has not been interpreted as `intrpt`?  In that case should there also be a test that `loads.scale` is `missing`?  `scale` is the 13th column so would be where the end of line comment could mistakenly end up.

Reply:
it's just checking that `loads` parse as expected (but checking only select columns as a proxy, rather than every column)

> scale is the 13th column so would be where the end of line comment could mistakenly end up.

i don't follow this - can you explain?

Anyway it sounds like if we want something like this PR then it _does_ need to parse commnts from more than just Buses data, so this'd need updating

Reply:
> scale is the 13th column so would be where the end of line comment could mistakenly end up.

I thought this test might be intended to check that the end of line comment hadn't been parsed and placed into the last column by mistake.  In the test file for this test, `loads` has 12 columns, so I thought checking the 13th column was `missing` would be informative.